R/api-functions.R

```diff
@@ -0,0 +1,79 @@
+# Data package identifier reservation ------------------------------------------
+#' Reserve EDI ID 
+#' @description This package returns a funder
+#' @param user_id A string that contains the user ID for the EDI data portal. If you 
+#' do not already have an EDI account you may create one on the EDI portal. \url {https://portal.edirepository.org/nis/login.jsp}
+#' @param password A string that contains the user password for the EDI data portal. 
+#' @return This function returns a edi identifier number. 
+#' @examples 
+#' reserve_edi_id(user_id = "samuelwright", 
```

Comment:
you should consider using .Renviron file for password example. you don't want to accidentally push a real password up to github.

Reply:
I do not use an example password but I do use Renviron to store password